### PR TITLE
1297 exterior lights schedule

### PR DESCRIPTION
**Pull request by kbenne**

### CODE REVIEW

**Comment by kbenne** [comment]
https://www.pivotaltracker.com/story/show/85579220

**Comment by kbenne** [comment]
This is for #1297

**Comment by kbenne** [comment]
doooh!! one second.  need to add the change log.

**Comment by macumber** [comment]
Haha, I was just gonna do that for you :-)

**Comment by kbenne** [comment]
I'm not sure i got the syntax correct.  There seems to be a slight difference between those items that you algorithmically added and those that are freeform.

File: CHANGELOG.md [hunk 1]
```diff
@@ -64,3 +64,4 @@
 [fixed]: http://nrel.github.io/OpenStudio-user-documentation/img/fixed.svg "Fixed"
 [improved]: http://nrel.github.io/OpenStudio-user-documentation/img/improved.svg "Improved"
 [removed]: http://nrel.github.io/OpenStudio-user-documentation/img/removed.svg "Removed"
+[changed]: ExteriorLights::schedule now returns boost::optional<Schedule> instead of Schedule
```
Comment (macumber) [inline CHANGELOG.md]: Ah yeah :clap:

**Comment by macumber** [comment]
@kbenne looks good I'll just wait for the robots to chime in

File: CHANGELOG.md (hunk 1 above)
Comment (macumber) [inline CHANGELOG.md]: Hmm oh yeah, this is wrong. Those links at the bottom are permanent, they import the images.  Your line should have gone at the top of the file

Comment (macumber) [inline CHANGELOG.md]: Under the changes in 1.6.3 (actually now I think your changes should be listed under a new 1.7.0 release).  Newest stuff is at the top of the changelog

**Pull request merged**